### ISSUE TITLE
Angular 10.1: Copilot leaves "style" on "body" element

### ISSUE THREAD

**Issue by ksilz** [leifermendez/ngx-copilot#4]
I'm using ngx-copilot with Angular 10.1. I use it to show two steps. After the last step, my code calls `removeWrapper()`. Still, ngx-copilot leaves some `style` code behind on my `<html>` element that wasn't there before the steps:

```
style="overflow: auto; --zoneY:179.375px; --zoneX:495.625px; --zoneColor: rgba(0, 0, 0, 0.7) ; --zoneSize:96.4063px;"
```

This code leads to a scrollbar appearing. Once I remove the `style` code, the scrollbar disappears. See below for a sample.

Is there a way for ngx-copilot to automatically remove the `style` code on my `<html>` element?

![Style on body left behind](https://user-images.githubusercontent.com/1061209/95196872-9fc4d100-07d0-11eb-82d4-59b8b95a8595.gif)

**Comment by leifermendez** [comment]
Hi @ksilz I'll check and comment

**Comment by ksilz** [comment]
Not a problem, I've got a workaround where I delete the `style` attribute myself.

**Comment by ksilz** [comment]
In unrelated new, can I take a crack at improving the README next week? I'm a technical writer, too - [here's my latest Baeldung article](https://www.baeldung.com/json-reduce-data-size).

**Comment by leifermendez** [comment]
Of course, it would be of great help to the community. I would appreciate it very much

**Comment by ksilz** [comment]
Ok, deal.
